docs/development/workflow/development_workflow.rst

```diff
@@ -609,10 +608,11 @@ exporting the final diff from a feature branch, then starting a new branch and
 applying only that patch.
 
 Many of us find that is it actually easiest to squash using rebase. In particular,
-you can rebase and squash within the existing branch using::
+you can rebase and squash within the existing branch using, where you have to replace
+``N`` in the command below with the number of commits you need to handle::
 
-  git fetch astropy
```

Comment:
Here, I typically use what was here rather than `HEAD~N`, since one might as well rebase while one squashes...

Reply:
I also think that the previous instructions were more useful.

Reply:
I felt like we had this conversation before but I must have forgotten about it. I will revert. 😆